src/physics/site_types/qubit.jl

```diff
@@ -331,6 +331,17 @@ function ITensors.op(::OpName"Ryy", ::SiteType"Qubit"; ϕ::Number)
 end
 ITensors.op(::OpName"RYY", t::SiteType"Qubit"; kwargs...) = op("Ryy", t; kwargs...)
 
+# Ising (XY) coupling gate
+function ITensors.op(::OpName"Rxy", t::SiteType"Qubit"; ϕ::Number)
```

Comment:
Wikipedia defines this as the complex conjugate of the definition you add here: https://en.wikipedia.org/wiki/List_of_quantum_logic_gates#Two_qubit_interaction_gates. I don't know what's standard but it seems like our other definitions are consistent with Wikipedia (besides the factor of 2 in the angles), and the Wikipedia definitions seem self-consistent.